Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js

```diff
@@ -120,13 +120,23 @@ window.asyncEditor.editor = {
         },
         submitEditor(nextEditorGroup) {
             const self = this;
-
-            self.api.submitEditor(
-                self.contentId,
-                self.editorGroup,
-                nextEditorGroup,
-                new FormData(self.$refs.editorForm),
-                (success, data) => { self.processApiData(success, data); });
+            const validationEvent = new CustomEvent('validationEvent', {

```

Comment:
Since there is nothing related to validation, it's more like a `submittingEditor` event. Also, I'd add a prefix to it, so it's distinguishable from other events. E.g., `asyncEditor.submittingEditor`.